webui/react/src/services/types.ts

```diff
@@ -14,6 +14,7 @@ export interface HttpApi<Input, Output>{
   httpOptions: (params: Input) => HttpOptions;
   postProcess?: (response: AxiosResponse<unknown>) => Output; // io type decoder.
   stubbedResponse?: unknown;
+  unAuthenticated?: boolean;
```

Comment:
**suggestion:** how about `needsAuth`?

Reply:
although it might make the logic a bit confusing. as in no longer `false` and null would do the same thing.

Reply:
I think I understand. Basically we are skewing to have authenticated endpoint as the default and opting IN to BE unauthenticated.

**super nit:** `unauthenticated` (all lowercase) just because its originally still one word.